scripts/templates/values.yaml

```diff
@@ -89,15 +89,15 @@ license:
 service:  #Nexus Repo NodePort Service
   nexus:
     name: nexus-repo-service
-    enabled: "false" #intentionally a string because it's used to configure operator via env variable
+    enabled: false
     type: NodePort
     protocol: TCP
     port: 80
     targetPort: 8081
   docker:  #Nodeport Service for Docker Service
     name: nexus-repo-docker-service
-    enabled: "false" #intentionally a string because it's used to configure operator via env variable
+    enabled: false
     type: NodePort
     protocol: TCP
     port: 9090
-    targetPort: 8081
+    targetPort: 9090
```

Comment:
is it correct?

Reply:
Yep. It's for docker connector. It's whatever the user specified as the http port if they enable docker http connector. We should probably add a comment as such